### PR TITLE
Code cleanup

### PR DESCRIPTION
**Pull request by tillkruss**

### CODE REVIEW

**Comment by tillkruss** [comment]
I left the description empty intentionally, because pet peeve 😉

File: src/Versioning.js [hunk 1]
```diff
@@ -5,7 +5,7 @@ class Versioning {
     /**
      * Create a new Versioning instance.
      *
-     * @param {object} manifest
+     * @param {object} Manifest
```
Comment (QWp6t) [inline src/Versioning.js]: The parameter is `manifest` not `Manifest`.

Comment (tillkruss) [inline src/Versioning.js]: Thanks, fixed!

**Comment by JeffreyWay** [comment]
haha - looks good. Thx!

**Pull request merged**